### DISCUSSION TITLE
NC AIO crashes when generating photo previews

### DISCUSSION THREAD

**Discussion by monsieurhannes** [nextcloud/all-in-one#1989] [Questions] [answered]
Hi,

I recently switched from NextcloudPi to AIO to have a more integrated Docker solution. NCP is more focused on an complete solution running the whole system.
Unfortunately I have some problems with the preview generator and I don't know why - as it was working in NCP, on the same hardware.

I installed Preview Generator and run it in the CLI, but after a while it takes the whole system down. I don't really know if it's due to the image size, but I already saw that the PHP-MEMORY-LIMIT is 512MB, which sounds more than the standard setting at 128MB, it should probably manage my 33 megapixel images, right?
It also doesn't necessarily seem to be the image size.

I'm currently on the beta-channel due to this issue: #1873 , could it be because of that? When I change to `latest`, the containers won't start...

## Server
Odroid N2 4GB RAM
Armbian Bullseye
Docker with Portainer
Nextcloud AIO v4.3.4

If you need any more information, I'm happy to provide it, but I'm not so experienced where to find things in the AIO-universe...

**Comment by szaimen** [comment, marked as answer]
Hi, this will be resolved with the update to nc26. You can follow https://github.com/nextcloud/all-in-one/issues/1846.

**Reply by monsieurhannes** [reply]
Thank you for your reply! Is there anything I can do to resolve it in the meantime?

**Reply by szaimen** [reply]
I don't think unfortunately. It is most likely crashing because too many previews get generated at the same time which causess the RAM to get filled and then everything stalling. Exactly this will get fixed with the update to 26.

**Reply by szaimen** [reply]
Hi, I just added some instructions how you can already upgrade to Nc26 if you want to try it out: https://github.com/nextcloud/all-in-one/issues/1846

**Reply by szaimen** [reply]
It looks like this may also be worth a try: https://github.com/nextcloud/server/issues/35643#issuecomment-1461687866
